diff --git a/setup.py b/setup.py
index f5e2cbb..b113ec0 100755
--- a/setup.py
+++ b/setup.py
@@ -96,16 +96,8 @@ setup(
         (['ordereddict'] if sys.version_info < (2, 7, 0) else []),
 
     tests_require=[
-        'beautifulsoup4',
-        'flask',
         'mock',
-        'pyechonest',
-        'pylast',
-        'rarfile',
-        'responses',
-        'pyxdg',
-        'pathlib',
-        'python-mpd2',
+        'responses'
     ],
 
     # Plugin (optional) dependencies:
