diff --git a/beets/ui/commands.py b/beets/ui/commands.py
index a772046..db114b0 100644
--- a/beets/ui/commands.py
+++ b/beets/ui/commands.py
@@ -1543,6 +1543,7 @@ def print_completion(*args):
 
 BASH_COMPLETION_PATHS = map(syspath, [
     u'/etc/bash_completion',
+    u'/etc/profile.d/bash-completion.sh',
     u'/usr/share/bash-completion/bash_completion',
     u'/usr/share/local/bash-completion/bash_completion',
     u'/opt/local/share/bash-completion/bash_completion',  # SmartOS
