--- deadbeef.desktop.in.orig	2013-12-08 16:34:09.000000000 -0500
+++ deadbeef.desktop.in	2014-02-07 17:06:19.270286167 -0500
@@ -13,29 +13,3 @@
 Type=Application
 Icon=deadbeef
 X-PulseAudio-Properties=media.role=music
-
-X-Ayatana-Desktop-Shortcuts=Play;Pause;Stop;Next;Prev
-[Play Shortcut Group]
-Name=Play
-Exec=deadbeef --play
-TargetEnvironment=Unity
-
-[Pause Shortcut Group]
-Name=Pause
-Exec=deadbeef --pause
-TargetEnvironment=Unity
-
-[Stop Shortcut Group]
-Name=Stop
-Exec=deadbeef --stop
-TargetEnvironment=Unity
-
-[Next Shortcut Group]
-Name=Next
-Exec=deadbeef --next
-TargetEnvironment=Unity
-
-[Prev Shortcut Group]
-Name=Prev
-Exec=deadbeef --prev
-TargetEnvironment=Unity
