diff --git a/Executables/Mathematica b/Executables/Mathematica
index d9a5530..d6bfb8b 100755
--- a/Executables/Mathematica
+++ b/Executables/Mathematica
@@ -3,8 +3,8 @@
 #  Mathematica 9.0 Front End command file
 #  Copyright 1988-2013 Wolfram Research, Inc.
 
-#  Make certain that ${PATH} includes /usr/bin and /bin
-PATH="/usr/bin:/bin:${PATH}"
+#  Make certain that ${PATH} includes /bin, /usr/bin, and /usr/sbin (lspci)
+PATH="/bin:/usr/bin:/usr/sbin:${PATH}"
 
 
 
