From 6583a1a632779e72ba6ecfa32a10f30980081876 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Thadd=C3=A4us=20Tintenfisch?= <thad.fisch@gmail.com>
Date: Sun, 23 Feb 2014 09:02:58 +0300
Subject: Fix segfault on session start

Signed-off-by: Eric Koegel <eric.koegel@gmail.com>

diff --git a/src/xfce-desktop.c b/src/xfce-desktop.c
index aff857e..b076589 100644
--- a/src/xfce-desktop.c
+++ b/src/xfce-desktop.c
@@ -1662,6 +1662,10 @@ xfce_desktop_refresh(XfceDesktop *desktop)
     if(!gtk_widget_get_realized(GTK_WIDGET(desktop)))
         return;
 
+    if(desktop->priv->workspaces == NULL) {
+        return;
+    }
+
     current_workspace = xfce_desktop_get_current_workspace(desktop);
 
     /* reload backgrounds */
-- 
cgit v0.10.1

